### DISCUSSION TITLE
直接拉取github仓库内容来同步

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Mobile#73] [General]
尝试了一下，发现同步效果并不太好。

感觉可以直接拉取github仓库内容来同步。但这样好像也需要跟太记桌面版一样通过github授权了。

**Comment by linonetwo** [comment]
应该不行，手机上不方便跑git。

还不如指望往 SoLiD 上同步，或者之后往 MWS 搭的类似 tiddlyhost 的服务上同步

**Reply by dongrentianyu** [reply]
好吧。

**Reply by linonetwo** [reply]
你要是想学 JS 了，可以看看 https://github.com/linonetwo/solid-tiddlywiki-syncadaptor 学习一下

**Reply by dongrentianyu** [reply]
好的。
